projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1138,6 +1132,69 @@ function wpcom_request_domains_list() {
 	return $decoded_body->domains;
 }
 
+/**
+ * Make a request to the WordPress.com API to get the values of the user
+ * attributes sent by the client.
+ *
+ * @param array $attributes The attributes to request.
+ * @return array|WP_Error Array of user attributes or WP_Error if the request fails.
+ */
+function wpcom_launchpad_request_user_attributes( $attributes ) {
+	// Use a static variable as a temporary in-memory cache to avoid multiple outbound
+	// HTTP requests within a single incoming request.
+	// We don't expect this to be triggered multiple times, but it's worth adding some
+	// light caching to avoid multiple, possibly slow HTTP requests where the underlying data
+	// is highly unlikely to change.
+	// The "cache" only lasts as long as the current request/memory space, so we don't need to invalidate it.
+	static $cached_attributes = null;
+
+	if ( null !== $cached_attributes ) {
+		return $cached_attributes;
+	}
+
+	if ( ! is_array( $attributes ) ) {
+		return new \WP_Error(
+			'invalid_attributes',
+			esc_html__( 'Invalid attributes.', 'jetpack-mu-wpcom' )
+		);
+	}
+
+	$query_params  = build_query( array( 'attributes' => $attributes ) );
+	$wpcom_request = Automattic\Jetpack\Connection\Client::wpcom_json_api_request_as_user(
+		'/jetpack-user-attributes?' . $query_params,
+		'v2',
+		array(
+			'method'  => 'GET',
+			'headers' => array(
+				'X-Forwarded-For' => ( new Automattic\Jetpack\Status\Visitor() )->get_ip( true ),
+			),
+		)
+	);
+
+	$response_code = wp_remote_retrieve_response_code( $wpcom_request );
+	if ( 200 !== $response_code ) {
+		return new \WP_Error(
+			'failed_to_fetch_data',
+			esc_html__( 'Unable to fetch the requested data.', 'jetpack-mu-wpcom' ),
+			array( 'status' => $response_code )
+		);
+	}
+
+	$body         = wp_remote_retrieve_body( $wpcom_request );
+	$decoded_body = json_decode( $body );
+
+	if ( ! is_array( $decoded_body->user_attributes ) && ! isset( $decoded_body->user_attributes ) ) {
```

Comment:
Shouldn't we do this the other way around? I think `isset()` will always be true if we have an array.
```suggestion
	if ( ! isset( $decoded_body->user_attributes ) || ! is_array( $decoded_body->user_attributes ) ) {
```

Reply:
🤦 Thanks for catching it!